src/SourceBuild/content/eng/build.ps1

```diff
@@ -66,6 +74,19 @@ function Build {
     @properties
 }
 
+function Test {
+  InitializeToolset
+
+  $bl = if ($binaryLog) { '/bl:' + (Join-Path $LogDir 'ScenarioTests.binlog') } else { '' }
+  $tp = if ($testProjects) { '/p:DotNetTestProjectsToRun="' + $testProjects + '"' } else { '' }
+
+  MSBuild $buildProj `
+    $bl `
+    /t:Test `
+    $tp `
+    @properties
+}
+
```

Comment:
Same comment as in build.sh. We shouldn't need a separate function and MSBuild invocation and instead just pass the Test property in same as Arcade's build scripts.

Reply:
These changes have been removed in https://github.com/dotnet/installer/pull/19222/commits/49c04fc531de986bab872236a4cb7a8b651be539 as part of integrating with the new test infra from https://github.com/dotnet/installer/pull/19290.